packages/pilot/src/containers/Sidebar/index.js

```diff
@@ -41,28 +41,35 @@ class SidebarContainer extends React.Component {
 
   renderSections () {
     const {
+      anticipationLimit,
       balance,
       onAnticipate,
       onWithdraw,
       t,
+      transfersPricing,
     } = this.props
 
     const getFrombalance = propOr(null, __, balance)
     const available = getFrombalance('available')
     const waitingFunds = getFrombalance('waitingFunds')
 
+    const disableBtnWithdraw = available <= transfersPricing.ted
+    const MINIMUM_WITHDRAWAL_VALUE = 100
+
     return (
       <SidebarSections
         sections={[
           {
             action: onWithdraw,
             actionTitle: t('pages.sidebar.withdraw'),
+            disabled: disableBtnWithdraw + MINIMUM_WITHDRAWAL_VALUE,
```

Comment:
complementando o outro comentário acho que aqui seria direto a validação
```suggestion
            disabled: disableBtnWithdraw
```

Reply:
ok!